component/encryptor_v2/encryptor_v2.go

```diff
@@ -0,0 +1,468 @@
+package encryptor_v2
```

Comment:
>encryptor_v2 [](http://example.com/codeflow?start=8&length=12)

instead of creating 2 packages can we have both in the same encryptor package? If intent is to eventually keep one then we need not split it in two for now. Also code like aes_encryption.go and any other code that can be shared need not be duplicated.